extensions/wikia/Recirculation/js/gruntfile.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function (grunt) {
```

Comment:
Do you still need this gruntfile and `.gitignore` above after going with `wikia.loader` for templates?